lms/templates/footer.html

```diff
@@ -83,3 +83,19 @@
     <link rel="stylesheet" type="text/css" href="${url}"></link>
   % endfor
 % endif
+
+% if settings.FEATURES.get('ENABLE_COOKIE_CONSENT', False):
+<!-- Cookie Consent -->
+<link rel="stylesheet" type="text/css" href="//cdnjs.cloudflare.com/ajax/libs/cookieconsent2/3.0.1/cookieconsent.min.css" />
+<script src="//cdnjs.cloudflare.com/ajax/libs/cookieconsent2/3.0.1/cookieconsent.min.js"></script>
+<script>
+  window.cookieconsent.initialise({
```

Comment:
If I'm understanding this correctly, the text will not be localized correctly. I think that you should explicitly pass all of the messages as translated strings. See https://silktide.com/tools/cookie-consent/docs/installation/ for a description as to how to do this.